docs/src/apps/jsonrpc-api.md

```diff
@@ -374,7 +374,7 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id":1,"m
 {"jsonrpc":"2.0","result":[5,6,7,8,9,10],"id":1}
 ```
 
-### getConfirmedSignaturesForAddress
+### getConfirmedSignaturesForAddressDepricated
```

Comment:
Or maybe just remove it entirely?

Reply:
If we're not going to play this safe, I prefer removal over renaming, so we don't have to remember to clean it up later.